metadata-ingestion/src/datahub/ingestion/source/sql/sql_config.py

```diff
@@ -99,7 +99,8 @@ class BasicSQLAlchemyConfig(SQLAlchemyConfig):
     host_port: str = Field(description="host URL")
     database: Optional[str] = Field(default=None, description="database (catalog)")
     database_alias: Optional[str] = Field(
-        default=None, description="Alias to apply to database when ingesting."
+        default=None,
+        description="[Deprecated] Alias to apply to database when ingesting.",
```

Comment:
Can we add the `pydantic_field_deprecated` here

Reply:
Its already there. However that doesn't mark the field as deprecated in docs, so I preferred to add it in field description to highlight in docs that this field is deprecated..